pkg/volume/flexvolume/mounter-defaults.go

```diff
@@ -43,7 +43,7 @@ func (f *mounterDefaults) SetUpAt(dir string, mounterArgs volume.MounterArgs) er
 
 // Returns the default volume attributes.
 func (f *mounterDefaults) GetAttributes() volume.Attributes {
-	klog.V(5).Infof(logPrefix(f.plugin), "using default GetAttributes")
+	klog.V(5).Info(logPrefix(f.plugin), "using default GetAttributes")
```

Comment:
Info or InfoS ?

Reply:
No plan to migrate to structed log in this PR.